test/briber.js

```diff
@@ -0,0 +1,287 @@
+/* eslint-disable no-undef */ // Avoid the linter considering truffle elements as undef.
+
+const Pinakion = artifacts.require('MiniMeTokenERC20')
+const ConstantNG = artifacts.require('ConstantNG')
+const Kleros = artifacts.require('Kleros')
+const Briber = artifacts.require('Briber')
+const tryCatch = require('./exception.js').tryCatch
+const errTypes = require('./exception.js').errTypes
+
+contract('Briber', function(accounts) {
+  const timePeriod = 0
+  const randomNumber = 10
+  const governor = accounts[0]
+  const juror1 = accounts[1]
+  const juror2 = accounts[2]
+  const juror3 = accounts[3]
+  const attacker = accounts[4]
+  const other = accounts[5]
+  const timePeriods = [
+    timePeriod,
+    timePeriod,
+    timePeriod,
+    timePeriod,
+    timePeriod
+  ]
+  const bribe = 0.2e18

```

Comment:
Tests should be as specific as possible (contrary to classic code, they should not be modular and try to hardcode everything possible).

Reply:
Should I add tests that cover some extreme cases? Like many disputes and many appeals or bribe being exceedingly high?